crates/compiler/fmt/src/pattern.rs

```diff
@@ -578,7 +588,21 @@ pub fn pattern_fmt_apply(
         }
     }
 
-    if parens || use_commas_and_parens {
+    if use_commas_and_parens {
+        if is_multiline {
+            buf.ensure_ends_with_newline();
+            buf.indent(indent);
+        }
+        if buf.ends_with_newline() {
```

Comment:
Are there supposed to be two indentations here? Seems like they should be combined, or one should be removed.

Reply:
Hmmm....I don't know what happened here.  I think I'll do a fast follow to change this up to do what I did in `fmt/src/expr.rs` and use the collection utility to format this